sbi/neural_nets/factory.py

```diff
@@ -174,8 +183,26 @@ def build_fn(batch_theta, batch_x):
             return build_nsf(batch_x=batch_x, batch_y=batch_theta, **kwargs)
         elif model == "mnle":
             return build_mnle(batch_x=batch_x, batch_y=batch_theta, **kwargs)
+        elif model == "zuko_nice":
+            return build_zuko_nice(batch_x=batch_x, batch_y=batch_theta, **kwargs)
         elif model == "zuko_maf":
             return build_zuko_maf(batch_x=batch_x, batch_y=batch_theta, **kwargs)
+        elif model == "zuko_nsf":
+            return build_zuko_nsf(batch_x=batch_x, batch_y=batch_theta, **kwargs)
```

Comment:
These are not covered by tests.

Reply:
can be easily added to the ones in `tests/neural_nets_factory.py`